DesignSafe-CI/portal
# Task/des 2862 update form validation messages

## Overview: ##
add custom messages to required fields.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2862](https://tacc-main.atlassian.net/browse/DES-2862)

## Summary of Changes: ##
I updated error messages for the following forms:

- BaseProjectForm
- ProjectCategoryForm
- ProjectEntityForm

## Testing Steps: ##
1. Create a new project, leave fields blank, check the error messages
2. Once a project has been created, when editing it, leave fields blank, check the error messages.
3. For different project types, under Curation Directory, add an entity, leave fields blank, check the error messages.

## UI Photos:

![Screen Shot 2024-06-14 at 2 42 02 PM](https://github.com/DesignSafe-CI/portal/assets/35277477/2db7c30b-cc49-484b-951a-4d7966e8f373)
![Screen Shot 2024-06-14 at 2 41 27 PM](https://github.com/DesignSafe-CI/portal/assets/35277477/327eb2dc-fc47-4f39-8892-fda8250a9dd4)


## Notes: ##
